build.js

```diff
@@ -40,7 +40,10 @@ async function updateTranslations() {
 }
 
 (async() => {
-    await updateTranslations();
+    if (process.argv.slice(2)[0] !== '--skip-translations') {
```

Comment:
Should do the slicing above this statement (in case we add more flags in the future). Also should do a ".contains(...)" or similar type lookup.